templates/wechatgame/first-screen.js

```diff
@@ -177,8 +180,8 @@ function initProgressVertexBuffer() {
 }
 
 function updateVertexBuffer() {
-    const widthRatio = image.width / canvas.width;
-    const heightRatio = image.height / canvas.height;
+    const widthRatio = image.width / canvas.width * 1.35;
+    const heightRatio = image.height / canvas.height * 1.35;
```

Comment:
Why have the magic number?

Reply:
firstScreen is rendered according to the pixels of the provided image, and the size parameters of the display are adjusted according to the provided UI design drawing.

Reply:
Yep, it is better to add the information in codes. Or we will forget why need do it like this.

Reply:
got it.